feature-group-definitions/transition-behavior.yml

```
@@ -1,4 +1,5 @@
 name: transition-behavior
+description: The `transition-behavior` CSS property specifies whether transitions will be started for properties whose animation behavior is discrete. Properties with a discrete animation type can't be interpolated and swap from their start value to the end value at 50%.
```

Since `transition-behavior: allow-discrete` is the only value other than the default, we can spell it out:

```suggestion
description: The `transition-behavior: allow-discrete` CSS property value allows transitions for properties whose animation behavior is discrete. Such properties can't be interpolated and swap from their start value to the end value at 50%.
```